src/main/java/org/cyclops/integrateddynamics/part/PartTypePanelDisplay.java

```diff
@@ -32,7 +33,7 @@ protected Block createBlock(BlockConfig blockConfig) {
 
     @Override
     public int getConsumptionRate(State state) {
-        return state.hasVariable() ? 2 : 1;
+        return GeneralConfig.panelDisplayBaseConsumption;
```

Comment:
Could you add `panelDisplayBaseConsumptionEnabled` and `panelDisplayBaseConsumptionDisabled` here?

Reply:
I've split the display panel config option into separate options for enabled and disabled states.